### PR TITLE
Tutorial documentation fixes 

### PR DESCRIPTION
**Pull request by akshaysridhar**
1) Agnesi HS/ NHS problems - LaTeX formatting / typos fixed 
2) RTB - Links to documentation and page build, external links fixed
3) Add RTB to list_of_tutorials.jl
	modified:   list_of_tutorials.jl
	modified:   ../tutorials/Atmos/agnesi_hs_lin.jl
	modified:   ../tutorials/Atmos/agnesi_nh_lin.jl
	modified:   ../tutorials/Atmos/risingbubble.jl

I have

- [ ] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [ ] Followed all necessary [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and run `julia .dev/climaformat.jl .`
- [ ] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [x] There are no open pull requests for this already
- [x] CLIMA developers with relevant expertise have been assigned to review this submission
- [x] The code conforms to the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [x] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

File: tutorials/Atmos/risingbubble.jl [hunk 1]
```diff
@@ -1,13 +1,12 @@
-# # [Rising Thermal Bubble](@id EX-RTB-docs)
+# # [Rising Thermal Bubble]
```
Comment (jiahe23) [inline tutorials/Atmos/risingbubble.jl:1]: could you remove the brackets here?

File: docs/list_of_tutorials.jl [hunk 2]
```diff
@@ -26,6 +26,8 @@ if generate_tutorials
                 "Atmos/agnesi_hs_lin.jl",
             "Linear Non-Hydrostatic Mountain (Topography)" =>
                 "Atmos/agnesi_nh_lin.jl",
+            "Dry Rising Thermal Bubble (Basic LES Experiment)" =>
```
Comment (jiahe23) [inline docs/list_of_tutorials.jl:29]: how about "Basic LES Experiment (Dry Rising Thermal Bubble)"?

File: tutorials/Atmos/risingbubble.jl (hunk 1 above)
Comment (akshaysridhar) [reply tutorials/Atmos/risingbubble.jl]: fixed

**Review by jiahe23 — APPROVED** [review]

**Pull request merged**